docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -78,6 +78,9 @@ Then you can create a `.env` file or directly use the deployment platform you us
 
 **Path —** `.env`.
 
+::: danger When deploying to production please ensure you set your JWT secret in your ENV. Read more [here](https://strapi.io/documentation/developer-docs/latest/setup-deployment-guides/configurations.html#server)
```

Comment:
```suggestion
::: danger When deploying to production please ensure you set your JWT secret in your ENV (see [server configuration](https://strapi.io/documentation/developer-docs/latest/setup-deployment-guides/configurations.html#server)).
```

Reply:
@pwizla I think this follows the WIP doc style guide that @meganelacheny has been working on.

Reply:
Exactly! Thank you Derrick :-)

Reply:
I would also change "in your ENV" for "in your `.env` file". Apart from that, looks good to me, thank you for you contribution @Cpaczek :-)

Reply:
updated my suggestion to match that.